project/Dependencies.scala

```diff
@@ -6,6 +6,7 @@ object Dependencies {
   private val akkaV = "2.5.32" // scala-steward:off (CROM-6637)
   private val ammoniteOpsV = "2.4.1"
   private val apacheHttpClientV = "4.5.13"
+  private val apacheHttpClient5V = "5.3.1"
```

Comment:
Why do we need two versions? Are the other usages not compatible with 5.x?

Reply:
The published bard client uses v5, which is the standard for Spring Boot 3 services.

Reply:
I believe you, what I'm saying is, if you make our current stuff also use v5, does that work?

Reply:
I was able to get this working